test/robot/go_robot_go.sh

```diff
@@ -0,0 +1,320 @@
+#!/bin/bash
+shopt -s extglob
+clear
+pushd $(dirname $0/../..) > /dev/null
+OUTDIR="bazel-out/robot/"
+BUILD_BOT_DIR="${OUTDIR}build_bot/"
```

Comment:
Typically you don't put the final `/` into the variable. `$OUTDIR/foo` is much more readable and understandable than `${OUTDIR}foo`